### ISSUE TITLE
Errors building with vue-tsc --noEmit

### ISSUE THREAD

**Issue by zachad** [tailwindlabs/heroicons#318]
I'm running into typescript errors on every single component when trying to build a project with `vite` and `vue-tsc` (using Vue3 and TypeScript):

Each component throws these two errors:

```
node_modules/@heroicons/vue/solid/AcademicCapIcon.d.ts(1,16): error TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.
```
```
node_modules/@heroicons/vue/solid/AcademicCapIcon.d.ts(1,30): error TS2339: Property 'DefineComponent' does not exist on type 'Promise...
```

Running `vite` in dev mode works as expected. This only happens when running `vue-tsc --noEmit`. 

Has anybody else run into this, or found a solution?

**Comment by DrevanTonder** [comment]
Yup, same when using `vite` and `vue-tsc --noEmit`. Works find in dev mode.

**Comment by zmwangx** [comment]
PR in #322.

**Comment by bradlc** [comment]
Thanks for reporting. Should be fixed in `v1.0.4` 👍
